.gitignore

```diff
@@ -3,3 +3,7 @@
 /target
 /RemoteSystemsTempFiles
 /bin
+/src/main/resources/application.yaml
```

Comment:
Why are we ignoring configuration files?

Reply:
fixed